jekyll/_cci2/browser-testing.md

```diff
@@ -173,78 +175,20 @@ build:
 
 ### LambdaTest Browser Testing Orb Example
 
-LambdaTest has developed a CircleCI orb for browser compatibility testing that enables you to open a Lambda Tunnel before performing any browser testing. Also, to ease the process of integration between LambdaTest & CircleCI. This orb (a package of configurations that you can use in your workflow) has been developed and certified for use and can simplify your configuration workflows. An example of the orb is shown below.
+LambdaTest has developed a [CircleCI orb](https://circleci.com/orbs/registry/orb/lambdatest/lambda-tunnel) for browser compatibility testing that enables you to open a Lambda Tunnel before performing any browser testing, easing the process of integrating LambdaTest with CircleCI. Use the orb to quickly set up a Lambda tunnel and the define your test steps
+
 {% raw %}
-```
```

Comment:
This was just a straight copy of the orb source. replacing with a usage example...